back/spec/policies/cosponsors_initiative_policy_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CosponsorsInitiativePolicy do
+  subject(:policy) { described_class.new(user, cosponsors_initiative) }
+
+  let(:scope) { CosponsorsInitiativePolicy::Scope.new(user, CosponsorsInitiative) }
```

Comment:
The same here. This let variable doesn't seem to be used.

Reply:
Removed in #5578